Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -33,14 +33,20 @@ class Navigator {
     }
 
     private lazy var routeCoordinator: RoutesCoordinator = {
-        .init(mainRouteSetupHandler: { [weak self] route, legIndex, completion in
-            self?.navigator.setPrimaryRouteForRoute(route, legIndex: legIndex) { [weak self] result in
+        .init(routesSetupHandler: { [weak self] route, legIndex, alternativeRoutes, completion in
+            var routesParams: SetRoutesParams? = nil
+            if let route = route {
+                routesParams = SetRoutesParams(primaryRoute: route,
+                                               legIndex: legIndex,
+                                               alternativeRoutes: alternativeRoutes)
+            }
+            
+            self?.navigator.setRoutesFor(routesParams) { [weak self] result in
                 if result.isValue() {
-                    let routeInfo = result.value!
```

Comment:
Was it removed? Why do we use `route` here instead of `result`?

Reply:
That's how NN updated it. According to the latest changes, they are injecting RouteInfo into the original `RouteInterface` object. Result value now reports only alternative routes data, which we don't need on that stage